templates/login.mako

```diff
@@ -0,0 +1,28 @@
+<%!
+    from json import dumps, loads
+    from flask import g, request
+    from copyvios.misc import cache
+%>\
+<%include file="/support/header.mako" args="title='Login | Earwig\'s Copyvio Detector', splash=True"/>
+% if error:
+    <div id="info-box" class="red-box">
+        <p>Error trying to log in: ${error}</p>
```

Comment:
Use `${error | h}` to prevent XSS?

Reply:
Didn't realize I missed it here. Nice catch!